hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -725,7 +728,7 @@ aggregate st = \case
         { parameters = parameters
         , pendingCommits = Set.fromList parties
         , committed = mempty
-        , chainState
+        , chainState = chainState
```

Comment:
Minor: no need to put an assignment here and below. The punning (as it was before) would update the field already.